### PR TITLE
Extract telegram bot code to separate file

### PR DESCRIPTION
**Pull request by brawaga**
Extracted.

### CODE REVIEW

File: self_therapy/selfcare/management/commands/start_care.py [hunk 1]
```diff
@@ -18,91 +18,6 @@ class Command(BaseCommand):
...
     def handle(self, *args, **options):
         """Make a bot and start work."""
-        bot = TeleBot(token=BOT_TOKEN)
-        start_keyboard = types.ReplyKeyboardMarkup(resize_keyboard=True)
-        # start_keyboard = types.InlineKeyboardMarkup()
-        btn_create_user = types.KeyboardButton('/createuser')
-        btn_loadpreset = types.KeyboardButton('/loadpreset')
-        btn_getemotion = types.KeyboardButton('/getemotion')
-        btn_getsupport = types.KeyboardButton('/getsupport')
-        # btn_addsupport = types.InlineKeyboardButton('Можно русским языком',
```
Comment (belyashnikovatn) [inline self_therapy/selfcare/management/commands/start_care.py:28]: whats this? lets delete

File: self_therapy/selfcare/telegram_bot.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from typing import Callable
+
+from telebot import TeleBot, types, util
+from selfcare.models import Person, PersonsHelpTip, PresetHelpTip, Emotion
+
+
+class TelegramBot:
```
Comment (belyashnikovatn) [inline self_therapy/selfcare/telegram_bot.py:9]: need docstrings

**Review by belyashnikovatn — CHANGES_REQUESTED** [review]
see comments

File: self_therapy/selfcare/telegram_bot.py (hunk 2 above)
Comment (brawaga) [reply self_therapy/selfcare/telegram_bot.py:9]: Ok, done.

**Review by belyashnikovatn — APPROVED** [review]
LGTM! Good job

**Comment by brawaga** [comment]
> LGTM! Good job

Thanks!

**Pull request merged**